src/server/config/express.js

```diff
@@ -55,6 +56,14 @@ export default (server) => {
     proxyOneQuran.web(req, res);
   });
 
+  server.use('/apple-app-site-association', (req, res) => {
```

Comment:
apple dictates it cannot be a `.json` file and need to be served with `application/pkcs7-mime'` content-type